wire-library/wire-grpc-mockwebserver/src/main/java/com/squareup/wire/mockwebserver/GrpcDispatcher.kt

```diff
@@ -155,7 +156,7 @@ class GrpcDispatcher(
     val source = GrpcMessageSource(
         source = request.body,
         messageAdapter = protoAdapter,
-        grpcEncoding = request.headers["grpc-encoding"]
+        codec = GrpcCodec.IdentityGrpcCodec
```

Comment:
Why don't we read it anymore?

Reply:
The client supplies the codec for decoding and encoding as mentioned in my description.

Reply:
From gRPC [documentation](https://www.google.com/url?sa=t&source=web&rct=j&url=https://github.com/grpc/grpc-go/blob/master/Documentation/encoding.md&ved=2ahUKEwielYGrnZvwAhVQhf0HHTVYBb4QFjAAegQIBRAC&usg=AOvVaw0SlY9CscYcxEOqSyk2lG0K)

> Codecs, by definition, must be symmetric, so the same desired Codec should be registered in both client and server binaries.